build/configure

```diff
@@ -150,17 +150,23 @@ if [ $combined_packages -ne 0 ]; then
 	x86_64 )
             openssl098_libdir=/usr/local_ssl_0.9.8/lib
             openssl100_libdir=/usr/local_ssl_1.0.0/lib64
+            openssl110_libdir=/usr/local_ssl_1.1.0/lib
             ;;
 	
 	* )
             openssl098_libdir=/usr/local_ssl_0.9.8/lib
             openssl100_libdir=/usr/local_ssl_1.0.0/lib
+            [ `uname -m` = "x86_64" ] && openssl110_libdir=/usr/local_ssl_1.1.0/lib
```

Comment:
Ha, that's funny! You have the identical bug that you pointed out to me for OMI! 😃 

So, in what code path would line 159 actually run?

Reply:
I have copied it from your code, thought it is handling some corner case. I will remove this line.

Reply:
Fixed